### DISCUSSION TITLE
Talks from other events

### DISCUSSION THREAD

**Discussion by plaindocs** [pretalx/pretalx#1133] [Ideas]
In the review view you show other talks by the same speaker, which is super useful. 
It is probably our fairly niche use, but it would be super useful to show talks from other events by the same speaker as well (maybe with an opt out, or a within X years type config)

**Comment by rixx** [comment]
This really is fairly niche, because even organisers with multiple years of events usually have slightly different teams each year, so possibly not all reviewers could see the same data there. I'd be more than willing to merge a PR implementing this with proper safeguards, but I can't make it a priority right now.

**Reply by rixx** [reply]
This is still not implemented, but at least there is now an organiser-level speaker directory where you can see past speakers and filter them on a per-event basis.

At this point, I’m not sure that this feature will ever be built, though I have to admit that it would be nice – it’s just that the permission checks are costly, making the feature fairly complex. If we do build it, we ought to fetch the additional speakers in a separate request to decouple the (potentially very expensive) checks from the more important initial page load.

**Reply by plaindocs** [reply]
Interesting, I'd not considered the permission aspect of it. I'd be happy for it to be admin level only, and therefor assume full visibility for all reviews. 

In practical terms we do this already outside of pretalx using the API.
